app/src/ui/resizable/resizable.tsx

```diff
@@ -47,6 +47,8 @@ export class Resizable extends React.Component<IResizableProps, {}> {
 
     document.addEventListener('mousemove', this.handleDragMove)
     document.addEventListener('mouseup', this.handleDragStop)
+
+    e.preventDefault()
```

Comment:
This is just a curiosity question: is there any time where we would want to call `e.preventDefault` first? I've noticed we do that in some places but that isn't being done here.

Reply:
It depends a lot on how the method is used. Usually it's better to put them up top but it only really matters if you call another method passing it the event or if you return early. If you have an escape hatch somewhere to bail out if some condition doesn't match then you shouldn't call preventDefault before. With methods like these it's a taste thing. My reasoning for putting them at the bottom here was that I wanted the methods to succeed before preventing the default action but either or would have worked just fine.